guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,501 @@
+{
```

Comment:
<p>(1)<s>Here either</s> <strong>Here,</strong> the variables could be either continuous</p><p>(2) But once the model is trained using a set of rasters with a certain cell <s>sizes </s> <strong>size </strong>for the different input layers</p><p>(3) <em>Handling polygons instead of </em><strong><em>the</em></strong><em> point</em></p><p><em>(4) </em>When <strong>the</strong> input is polygon for training <s>then</s> <strong>than</strong> for the raster it takes the mean of cells in case of continuous and <strong>the</strong> majority in case of continuous</p><p>(5) In the next <strong>version,</strong> It would be enhanced to min, max, sum, <strong>unique,</strong> etc of the values from the underlying rasters.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>